src/toast/ops/mapmaker.py

```diff
@@ -637,6 +647,18 @@ def _exec(self, data, detectors=None, **kwargs):
 
         self._log_info(comm, rank, "  finished final binning in", timer=timer)
 
+        # Write the outputs
+        # FIXME:  This all assumes the pointing operator is an instance of the
+        # PointingHealpix class.  We need to generalize distributed pixel data
+        # formats and associate them with the pointing operator.
+        if self.output_dir is not None:
```

Comment:
Big thumbs up for mapmaker writing out the maps. Yes, not as flexible as doing it in the pipeline but will eliminate so many lines of repeated code.